utils/test/accountUtils.ts

```diff
@@ -38,3 +57,101 @@ export const getEthBalance = async (account: Address): Promise<BigNumber> => {
 export const getWallets = async (): Promise<SignerWithAddress[]> => {
   return (await ethers.getSigners()) as SignerWithAddress[];
 };
+
+const getForkedDependencyAddresses = (): any => {
+  return {
+    whales: [
+      dependencies.DAI_WHALE,
+      dependencies.WETH_WHALE,
+      dependencies.WBTC_WHALE,
+      dependencies.USDC_WHALE,
+      dependencies.STETH_WHALE,
+    ],
+
+    tokens: [
+      dependencies.DAI[1],
+      dependencies.WETH[1],
+      dependencies.WBTC[1],
+      dependencies.USDC[1],
+      dependencies.STETH[1],
+    ],
+  };
+};
+
+// Mainnet token instances connected their impersonated
+// top holders to enable approval / transfer etc.
+export const getForkedTokens = (): ForkedTokens => {
+
+  // (eslint is confused by typescript enum keyword)
+  // eslint-disable-next-line no-unused-vars
+  const enum ids { DAI, WETH, WBTC, USDC, STETH }
+  const { whales, tokens } = getForkedDependencyAddresses();
+
+  const forkedTokens = {
+    dai: IERC20__factory.connect(tokens[ids.DAI], provider.getSigner(whales[ids.DAI])),
+    weth: IERC20__factory.connect(tokens[ids.WETH], provider.getSigner(whales[ids.WETH])),
+    wbtc: IERC20__factory.connect(tokens[ids.WBTC], provider.getSigner(whales[ids.WBTC])),
+    usdc: IERC20__factory.connect(tokens[ids.USDC], provider.getSigner(whales[ids.USDC])),
+    steth: IERC20__factory.connect(tokens[ids.STETH], provider.getSigner(whales[ids.STETH])),
+  };
+
+  return forkedTokens;
+};
+
+function toRpcQuantity(x: NumberLike): string {
+  let hex: string;
+  if (typeof x === "number" || typeof x === "bigint") {
+    // TODO: check that number is safe
+    hex = `0x${x.toString(16)}`;
+  } else if (typeof x === "string") {
+    if (!x.startsWith("0x")) {
+      throw new Error(
+        "Only 0x-prefixed hex-encoded strings are accepted"
+      );
+    }
+    hex = x;
+  } else if ("toHexString" in x) {
+    hex = x.toHexString();
+  } else if ("toString" in x) {
+    hex = x.toString(16);
+  } else {
+    throw new Error(
+      `${x as any} cannot be converted to an RPC quantity`
+    );
+  }
+
+  if (hex === "0x0") return hex;
+
+  return hex.startsWith("0x") ? hex.replace(/0x0+/, "0x") : `0x${hex}`;
+}
+
+/**
+ * Sets the balance for the given address.
+ *
+ * @param address The address whose balance will be edited.
+ * @param balance The new balance to set for the given address, in wei.
+ */
+export async function setEthBalance(
+  address: string,
+  balance: NumberLike
+): Promise<void> {
+  const provider = network.provider;
+  const balanceHex = toRpcQuantity(balance);
+  await provider.request({
+    method: "hardhat_setBalance",
+    params: [address, balanceHex],
+  });
+}
+
+export const initializeForkedTokens = async () => {
```

Comment:
I think it is best that we use the hardhat built-in rpc call to set balances rather than doing the hacky transfer eth and destruct a contract. Let me know if you have any questions or if maybe I overlooked something.

Reply:
Didn't know there was such an rpc call, but will check if I can use it here. Would probably be easier and allow us to remove the force funder contract.

Reply:
Just noticed that you had already commited that change, so I'm just using your version 👍